src/Mappers/Root/RootTypeMapperFactoryContext.php

```diff
@@ -26,6 +30,7 @@ public function __construct(
         private readonly RecursiveTypeMapperInterface $recursiveTypeMapper,
         private readonly ContainerInterface $container,
         private readonly CacheInterface $cache,
+        private readonly iterable $typeNamespaces,
```

Comment:
It'd be nice to have some validation for an iterable here.  Is there any reason why we shouldn't do this?

Reply:
What do you have in mind? I am not sure I understand what you are referring to.

Reply:
Seeing as `$typeNamespaces` could be `['foo', 1, Bar::class]`, it'd be nice to validate that input to ensure it's proper namespaces.

Reply:
Right. I don't think that's necessary here as we're dealing with a class that is only ever instantiated from internal code aka the lib's SchemaFactory. We have a couple of other places where we just rely on the iterable type hint and the IDE's extended type checking capabilities. I don't wanna add a runtime check where the statical analysis of the IDE should be sufficient.